css/style.css

```diff
@@ -0,0 +1,63 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: url('https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg');
+  padding: 50px 0;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#title,
+#description {
+  padding-bottom: 15px;
+  text-align: center;
+  color: rgb(27, 13, 59);
```

Comment:
Change required:
Your texts in the heading section (title and description ids) don't contrast enough with the background image. It's suggested to change the color in order to improve readability.

Reply:
- [ ] Brillant work you've done so far 👏 . To make this a step better and more professional, please use back strings to highlight tags and major elements when making reviews. This helps the user have clear visual differences in the sections mentioned and gives a professional setting for your review.

- For example, instead of  (title and description ids) above, you have  (`title` and `description` ids) 😃 . You can see the difference, and how it highlights the differences.
- Also, consider adding some friendly emojis to your reviews 😃 . There are times your reviews are valid but the students or project owner might not find it easy but  such emojis show a side of friendliness from your end on why such changes are necessary to make he/she a better developer 👍